README.md

```diff
@@ -293,6 +392,60 @@ We provide multiple datasets, both as starting points for each of the competitio
 
 We provide links to models pretrained using our dataset via the DCLM collection on Huggingface, found [here](https://huggingface.co/collections/mlfoundations/dclm-669938432ef5162d0d0bc14b). These models can be downloaded and evaluated using the OpenLM library.
 
+## Example Commands to Train 1B model with DCLM baseline
+Here we provide a quick set of commands for setting up, tokenize shuffle, training, and evaluation. For filtering, please see the relevant sections above. For specific details about the below commands, also refer to the relevant sections above.
+
+### Setup (with Conda)
+```bash
+conda create -n dclm python=3.10
+conda activate dclm
+
+git clone https://github.com/mlfoundations/DCLM.git
+cd DCLM
+export PYTHONPATH="$(pwd):$PYTHONPATH"
+
+pip install -r requirements.txt
+```
+
+### Tokenize Shuffle
+Here we include downloading DCLM data using AWS CLI.
+After running the below command, typically you would need to create a dataset json in exp_data/datasets/tokenized, but we provide one for you in this tutorial (exp_data/datasets/tokenized/dclm_gs3_ls1_rs_tokshuf.json).
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+bash rustup.sh -y
+source ~/.bashrc
+pushd rust_processing/tokshuf-rs
+cargo build --release
+
+aws s3 cp --recursive s3://commoncrawl/contrib/datacomp/DCLM-baseline/global-shard_03_of_10/local-shard_1_of_10/ dclm_local
+mkdir tokshuf_tmp
+mkdir dclm_tokshuf
+
+cargo run --release -- \
+--input dclm_local \
+--local-cell-dir tokshuf_tmp \ 
+--output dclm_tokshuf \
+--tokenizer "EleutherAI/gpt-neox-20b" \
+--seqlen 2049 \
+--wds-chunk-size 8192 \
+--num-local-cells 512
```

Comment:
Should `--num-local-cells` be equal to the number of available cores (in which case we should probably decrease the default / add a comment)

Reply:
No. Theres an explanation for this in the rust tokenize shuffle readme, but this default is reasonable.

Reply:
Ok sounds good!